app/controllers/api/scores_controller.rb

```diff
@@ -37,6 +37,16 @@ def destroy
       }
     end
 
+    def user_scores
+      score = Score.where(user_id: params[:user_id])
```

Comment:
Can we order them descending by their play date? Just like we do for the feed.

Don't forget about n+1 queries!

Reply:
Sure!